recipes/glpk/meta.yaml

```diff
@@ -11,10 +11,6 @@ source:
 
 build:
   number: 0
-  features:
-    - vc9      [win and py27]
```

Comment:
Yes, that was in your conda-recipes recipe but I remove when it was gaving me issues on my system.  I'll try to add it back it.